golang/lib/api_container_context/api_container_context.go

```diff
@@ -0,0 +1,36 @@
+package api_container_context
+
+type APIContainerContext struct {
+	containerID  string
+	ipInsideEnclave string
+	portInsideEnclave uint32
+	ipOnHostMachine string
+	portOnHostMachine uint32
+}
+
+func NewAPIContainerContext(containerID string, ipInsideEnclave string, portInsideEnclave uint32, ipOnHostMachine string, portOnHostMachine uint32) *APIContainerContext {
+	return &APIContainerContext{containerID: containerID, ipInsideEnclave: ipInsideEnclave, portInsideEnclave: portInsideEnclave, ipOnHostMachine: ipOnHostMachine, portOnHostMachine: portOnHostMachine}
+}
+
+func (A APIContainerContext) GetContainerID() string {
+	return A.containerID
+}
+
+func (A APIContainerContext) GetIpInsideEnclave() string {
```

Comment:
Let's uppercase the `Ip` to `IP` for consistency with Go style conventions (it's not really a big deal, only that it'll be an API break in the future)